lib/shared/widgets/extensions/route_extensions.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:io';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+extension RouteExtensions on Widget {
+  Route<dynamic> toRoute(BuildContext context) {
+    if (Platform.isIOS) {
```

Comment:
This specific page router is only needed to have a (native) page navigation animaiton, right? :)

We talked some weeks/months ago about having one behaviour for the app. So for example for spinners we wanna use the material (or custom) one and not native :)  Maybe for this navigation thing we also wanna do that? 🤔

Reply:
Just so I understand, you want to make sure that between gorouter and this extension we always use the same page animations?

Reply:
For now it has the exact same default page animation as the rest of the app/ gorouter. So I consider it resolved for now.